bin/init

```diff
@@ -66,6 +67,35 @@ setup_cpi() {
   fi
 }
 
+check_ldap() {
+  set +e
+
+  TIMEOUT=90
+  COUNT=0
+  RETRY=1
+
+  while [ $RETRY -ne 0 ]; do
+    case $(bundle exec rails r bin/check_ldap.rb | grep LDAP) in
+      "LDAP_DOWN")
```

Comment:
Shouldn't you have "LDAP_FAIL" here as well?

Reply:
See https://github.com/kubic-project/velum/pull/722#discussion_r253027250.

Reply:
I wasn't able to check if the authentication was working properly with my local debug.

Reply:
Well, I'd say it's a matter of deciding what it would make sense. Hopefully credentials should be alright, but if they aren't, then it means that at least the LDAP server is reachable, which is enough for this purposes. Hence ignore my comment :grin: